internal/util/errutil/errors.go

```diff
@@ -0,0 +1,37 @@
+package errutil
+
+import "github.com/cockroachdb/errors"
+
+type MultiErrors struct {
+	errs []error
+}
+
+func (e MultiErrors) Unwrap() error {
+	if len(e.errs) == 0 {
+		return nil
+	}
+	return MultiErrors{
+		errs: e.errs[1:],
+	}
+}
+
+func (e MultiErrors) Error() string {
+	final := e.errs[0]
```

Comment:
May cause panic if one `MultiErrors` is Unwrapped to zero errors.

Reply:
It should be, as the error of other errors lib (include std) panics if it's nil

Reply:
But you could construct a not nil but zero length  MultiErrors with Unwrap

Reply:
Make this type private done